pkgs/applications/gis/grass/clang-integer-conversion.patch

```diff
@@ -0,0 +1,21 @@
+diff -ur a/db/drivers/mysql/db.c b/db/drivers/mysql/db.c
```

Comment:
Who is the author of this patch ?

Reply:
I wrote it. I didn’t see anything upstream about a fix. I’m not sure how this is expected to work considering that the MySQL API expects an `unsigned int` not a string.

The patch isn’t upstreamed because it makes no attempt at fixing the issue portably. I think `strtonum` is available on Linux and other BSDs, but I don’t know about other platforms supported by GRASS. It would also bump the minimum required Darwin version to 11.0, which may not be acceptable. I went with `strtonum` because the alternatives are difficult or impossible to use safely.